v2/tools/generator/internal/astmodel/local_package_reference.go

```diff
@@ -67,6 +68,17 @@ func (pr LocalPackageReference) PackagePath() string {
 	return url
 }
 
+// ImportPath returns the path to use when importing this package
+func (pr LocalPackageReference) ImportPath() string {
+	return path.Join(pr.localPathPrefix, pr.group, pr.version)
+}
+
+// FolderPath returns the path to this package on disk.
+// rootFolder is the root folder for all our packages; if omitted, a relative path will be returned
```

Comment:
I don't see any reference to a `rootFolder`?

Reply:
Fixed up stale doc-comments.